sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DeleteRelationshipRequestOptions.java

```diff
@@ -1,29 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com.azure.digitaltwins.core.models;
-
-
-import com.azure.digitaltwins.core.DigitalTwinsClient;
-
-import java.util.List;
-
-/**
- * Optional settings that are specific to calls to {@link DigitalTwinsClient#updateComponent(String, String, List)} and its overloads.
- */
-public final class DeleteRelationshipRequestOptions extends DigitalTwinsRequestOptions {
```

Comment:
We get this type generated for free, so no need for this manually defined type anymore